.github/workflows/nx-cloud-agents.yml

```
@@ -131,7 +131,7 @@ jobs:
       - name: Get yarn cache directory path
         if: steps.package_manager.outputs.name == 'yarn'
         id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "dir=$(yarn cache dir)" >> GITHUB_OUTPUT
```

issue: the `$` is missing
```suggestion
        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```

---

🙃 - thanks for 👀 this.

I contemplated crafting something like a regex to do the find and replace, but thought I'd "do it live" instead. 🙊

---

I (actually GitHub Copilot) initially made the same mistake 🤷‍♂️